packages/database/src/runPostMigration.js

```diff
@@ -21,14 +21,14 @@ const EXCLUDED_TABLES_FROM_TRIGGER_CREATION = [
 // attempted
 const IMMUTABLE_TABLES = ['ancestor_descendant_relation'];
 
-const SELECT_TABLES_WITHOUT_TRIGGERS = `
+const getSelectForTablesWithoutTriggers = triggerSuffix => `
   SELECT t.table_name
     FROM information_schema.tables t
     LEFT JOIN information_schema.table_privileges privileges
       ON t.table_name = privileges.table_name
     WHERE NOT EXISTS (
       SELECT * from pg_trigger p
-      WHERE p.tgname LIKE concat(lower(t.table_name), '%', '_trigger')
+      WHERE p.tgname = concat(lower(t.table_name), '${triggerSuffix}')
```

Comment:
I guess that '%' was not required here? (Probably not, just double checking

Reply:
Yeah no longer required. We were previously checking for `entity_trigger` and `entity_immutable_trigger` using a single `LIKE` comparison, with the `%` a wildcard for `_immutable`. Now we explicitly check for each

Reply:
I see - thanks for the explanation! 🙌